src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class Square extends FigCol {
```

Comment:
Don't use verbs for class/interface names. The name `FigCol` suggests it might be a verb or a combination of verbs, which is not recommended for class names. Consider renaming it to something more indicative of what the class represents or its characteristics.